os/arch/arm/src/s5j/s5j_i2s.c

```diff
@@ -100,6 +100,10 @@
 #error CONFIG_AUDIO required by this driver
 #endif
 
+#ifndef CONFIG_S5J_I2S_MAXPORTS
```

Comment:
This is not configurable. Why do you add CONFIG_ prefix?

Reply:
If more devices are added in future , then this flag would be helpful. This variable should ideally be configurable from Kconfig. But currently we are yet to decide whether to add it Kconfig or not. If you want I can remove the CONFIG_ prefix.

Reply:
At that time, let's add prefix. We don't have any plan to add additional device. For that, we can't leave it as CONFIG_.